### ISSUE TITLE
missing asciidoctor attribute value for the name of an extension?

### ISSUE THREAD

**Issue by bashbaug** [KhronosGroup/OpenCL-Docs#1101]
There is one place in the OpenCL C spec that uses an extension name as an asciidoctor attribute (`cl_khr_integer_dot_product`):

```
ifdef::cl_khr_integer_dot_product[]
| {opencl_c_integer_dot_product_input_4x8bit_packed} +
  (when the {cl_khr_integer_dot_product} extension macro is defined)
| The OpenCL C compiler supports built-in functions that perform dot
products on 4x8 bit packed integer vectors.
```

This attribute is defined to get into this code section, but it is not currently defined to be anything, so in the rendered specification it looks like "(when the extension macro is defined)".  I don't think this is the expected behavior.

Should we define a value for this asciidoctor attribute?  Or should we just write the name of the extension manually?

**Comment by bashbaug** [comment]
Looking at this a bit more, I think this possibly should be the extension link `<<cl_khr_integer_dot_product>>` instead?

There are a few other places in the spec where we use similar text, for example:

> The functionality described in the following table [requires](https://github.com/KhronosGroup/OpenCL-Docs/issues/1101#unified-spec) support for the [cl_khr_subgroups](https://github.com/KhronosGroup/OpenCL-Docs/issues/1101#cl_khr_subgroups) extension macro; or for OpenCL 3.0 or newer and the __opencl_c_subgroups feature.

**Comment by oddhack** [comment]
Should be fixed in #1112. I think this was one of the first extensions I incorporated and my process improved over time.
